settings/src/lib.rs

```diff
@@ -147,6 +147,10 @@ pub struct NetworkSettings {
     /// The name of the device or router model
     #[serde(skip_serializing_if = "Option::is_none")]
     pub device: Option<String>,
+    /// The DER-encoded key used for HTTPS
```

Comment:
do we want to skip if none here? There's no default value so it will break if not configured right now.